Platform/CoffeelakeBoardPkg/Library/Stage1ABoardInitLib/Stage1ABoardInitLib.c

```diff
@@ -182,9 +254,10 @@ BoardInit (
   case PostTempRamInit:
     DisableWatchDogTimer ();
     EarlyPlatformDataCheck ();
+    EarlySioInit ();
     DebugPort = GetDebugPort ();
     if ((DebugPort != 0xFF) && (DebugPort < PCH_MAX_SERIALIO_UART_CONTROLLERS)) {
```

Comment:
Minor comment. The existing DebugPort != 0xFF code is not required. Everything else looks good to me.

Reply:
Right, will make the change